dependencies/pip/requirements.txt

```diff
@@ -92,7 +92,7 @@ python-dateutil==2.7.5
 python-digest==1.7
 pytz==2018.9
 pyxform==0.12.0
-redis==3.0.1              # via celery
+redis==3.2.0              # via celery
```

Comment:
It seems like we're asking for a newer version than specified by `celery`. That's OK, but if so, we should pin `redis==3.2.0` in `requirements.in`.